src/ocean/meta/codegen/CTFE.d

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+
+    Utilities for common CTFE operations needed for code generation
+
+    Copyright:
+        Copyright (c) 2017 sociomantic Labs GmbH. All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE_BOOST.txt for details.
+        Alternatively, this file may be distributed under the terms of the Tango
+        3-Clause BSD License (see LICENSE_BSD.txt for details).
+
+*******************************************************************************/
+
+module ocean.meta.codegen.CTFE;
+
+import ocean.meta.types.Qualifiers;
+
+/*******************************************************************************
+
+    Converts integer to string
+
+    Params:
+        i = integer to convert
+
+*******************************************************************************/
+
+istring toString ( long i )
```

Comment:
Please add a `ulong` version. Even if it's not used very often it's bad to not have it.